### PR TITLE
SK-X: Update gradle and dependencies

### PR DESCRIPTION
**Pull request by kupeliorhun**

### CODE REVIEW

File: buildSrc/src/main/kotlin/Deps.kt [hunk 1]
```diff
@@ -1,38 +1,39 @@
 object Versions {
 
-    const val kotlin = "1.3.50"
-    const val coroutines = "1.3.0"
+    const val kotlin = "1.6.2"
```
Comment (balazstothofficial) [inline buildSrc/src/main/kotlin/Deps.kt]: Why just Kotlin 1.6 and not 1.8?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,6 +1,6 @@
-#Mon Apr 01 15:38:24 CEST 2019
+#Thu Jun 01 14:36:42 CEST 2023
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
```
Comment (balazstothofficial) [inline gradle/wrapper/gradle-wrapper.properties]: Why not the newest gradle version?

File: settings.gradle [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
-        mavenCentral()
         google()
+        mavenCentral()
         maven { url 'https://repository.quickbirdstudios.com/repository/public' }
     }
 
     dependencies {
         classpath("com.quickbirdstudios:gradle-publishing:0.1.0")
+        classpath("com.android.tools.build:gradle:7.4.2")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.20")
```
Comment (balazstothofficial) [inline settings.gradle]: Nit: We should probably use the new way of importing plugins

File: buildSrc/src/main/kotlin/Deps.kt (hunk 1 above)
Comment (kupeliorhun) [reply buildSrc/src/main/kotlin/Deps.kt]: ✅

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (kupeliorhun) [reply gradle/wrapper/gradle-wrapper.properties]: ✅

File: settings.gradle (hunk 3 above)
Comment (kupeliorhun) [reply settings.gradle]: plugins in `build.gradle` files has been updated. Do you mean sth else?

✅

**Review by balazstothofficial — APPROVED** [review]

**Pull request merged**